apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -662,6 +661,10 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     end
   end
 
+  defdelegate determine_standard_challenge_queries(request, state), to: ExitProcessor.StandardExitChallenge
```

Comment:
one thing that `defdelegate` brings is clear semantics over API and implementation. I think that was the intention here as well

Reply:
TBH I wasn't really into it very much. It might be even cleaner to just have `ExitProcessor.StandardExitChallenge.determine_queries/determin_exit_txbytes/create`. I.e. without the `Core` stop at all.

But let me tackle this more thoroughly towards the end of these refactors, when organization of code into modules is going to happen.